### PR TITLE
Adding Mantis Publisher to the mantis helm chart

### PR DESCRIPTION
**Pull request by sundargates**
Ensure the volumes are persisted on the disk so that control plane restarts can happen without state loss.

### CODE REVIEW

File: mantis-stack/templates/_helpers/common.tpl [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{{/*
+Define the image configs for airflow containers
```
Comment (calvin681) [inline mantis-stack/templates/_helpers/common.tpl:2]: fix comment

**Review by calvin681 — APPROVED** [review]

**Comment by calvin681** [comment]
BTW, the PR title doesn't match the content of this PR.

File: mantis-stack/templates/_helpers/common.tpl (hunk 1 above)
Comment (sundargates) [reply mantis-stack/templates/_helpers/common.tpl:2]: Good point.

File: mantis-stack/values.yaml [hunk 2]
```diff
@@ -1,2 +1,20 @@
...
+  clusterDomain: "cluster.local"
+
+  ######################################
+  ## COMPONENT | Mantis Sample Publisher
+  ######################################
+  samplePublisher:
+    ## if the publisher sample deployment should be created
+    ##
+    enabled: true
+    image: localhost:5001/netflixoss/mantis-publisher-sample:latest
```
Comment (sundargates) [inline mantis-stack/values.yaml:20]: Fix the image

**Pull request merged**